src/pages/home/report/ContextMenu/PopoverReportActionContextMenu.js

```diff
@@ -258,6 +258,7 @@ class PopoverReportActionContextMenu extends React.Component {
                     onModalHide={this.runAndResetOnPopoverHide}
                     anchorPosition={this.state.popoverAnchorPosition}
                     animationIn="fadeIn"
+                    disableAnimation={false}
```

Comment:
Why does this need to be set to false here?

Reply:
I think we want to keep the animation for context menu. Let me otherwise.

Reply:
Ah for delete comment etc I see, I agree that is fine for the context menu